source/extensions/common/wasm/context.cc

```diff
@@ -1209,22 +1215,28 @@ Http::FilterHeadersStatus Context::onRequestHeaders() {
   return Http::FilterHeadersStatus::StopIteration;
 }
 
-Http::FilterDataStatus Context::onRequestBody(int body_buffer_length, bool end_of_stream) {
+Http::FilterDataStatus Context::onRequestBody(bool end_of_stream) {
   if (!wasm_->on_request_body_) {
     return Http::FilterDataStatus::Continue;
   }
   DeferAfterCallActions actions(this);
+  const auto buffer = getBuffer(BufferType::HttpRequestBody);
+  const auto body_len = (buffer == nullptr) ? 0 : buffer->length();
```

Comment:
Style: `body_len` => `buffer_length`, otherwise we use 2 terms to describe the same buffer.